charts/illumidesk/values.yaml

```diff
@@ -275,7 +275,7 @@ jupyterhub:
     image:
       # The JupyterHub image namespace. If the docker registry should be specified if using something other than
       # DockerHub (docker.io).
-      name: illumidesk/jupyterhub
+      name: "illumidesk/k8s-hubs"
```

Comment:
This should be [illumidesk/k8s-hub](https://hub.docker.com/r/illumidesk/k8s-hub) (without the s).